apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -127,6 +127,13 @@ type OpenStackControlPlaneSpec struct {
 	// Rabbitmq - Parameters related to the Rabbitmq service
 	Rabbitmq RabbitmqSection `json:"rabbitmq,omitempty"`
 
+	// +kubebuilder:validation:Optional
+	// NotificationsBusInstance - the name of RabbitMQ Cluster CR to select a Messaging
+	// Bus Service instance used by all services that produce or consume notifications.
+	// Avoid colocating it with RabbitMQ services used for PRC.
+	// That instance will be pushed down for services, unless overriden in templates.
+	NotificationsBusInstance *string `json:"notificationsBusInstance,omitempty"`
```

Comment:
Tested locally via ` OPENSTACK_IMG=quay.io/fpantano/openstack-operator-index:v0.3.3`.
If the field does not exist in the CR (is not specified), the CR is not changed, and this won't trigger any reconciliation loop in a minor update scenario. For this reason, having a `*string` ensures it is fully backward compatible with the previous versions.
Like we did for `Topologies`, if this field exists as an override, the top-level is ignored, otherwise it will be inherited by the top-level CR.